src/server.h

```diff
@@ -1148,6 +1149,10 @@ struct redisServer {
     size_t stat_module_cow_bytes;   /* Copy on write bytes during module fork. */
     uint64_t stat_clients_type_memory[CLIENT_TYPE_COUNT];/* Mem usage by type */
     long long stat_unexpected_error_replies; /* Number of unexpected (aof-loading, replica to master, etc.) error replies */
+    uint64_t stat_threaded_read_processed; /* Number of read events processed by IO threads */
+    uint64_t stat_threaded_write_processed; /* Number of write events processed by IO threads */
+    uint64_t stat_main_thread_read_processed; /* Number of read events processed by main threads */
+    uint64_t stat_main_thread_write_processed; /* Number of write events processed by main threads */
```

Comment:
I would convert these to long longs so that the printf is printing the correct type. (You can see in the ubuntu build it throws warnings since it's a different type)